config/crd/bases/v1/_.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.16.3
+spec:
+  group: ""
+  names:
+    kind: ""
+    plural: ""
+  scope: ""
+  versions: null
```

Comment:
This looks weird. Could the new allowlist synchronizer be creating this? If you delete this file and run `make generate` again, does this file get recreated?

Reply:
I thinks this will try to generate CRD  because it references TypeMeta, ObjectMeta and is under paths passed to the controller-gen. Simples fix would be to move it under `pkg`
https://github.com/DataDog/datadog-operator/blob/main/Makefile#L144-L145